src/java.base/share/classes/jdk/internal/misc/VM.java

```diff
@@ -482,6 +483,19 @@ public static boolean isSetUID() {
      */
     public static native boolean isCDSSharingEnabled();
 
+    /**
+     * Check if DumpLoadedClassList is set and the file is open.
+     */
+    public static boolean isDumpLoadedClassListSetAndOpen;
+    private static native boolean isDumpLoadedClassListSetAndOpen0();
```

Comment:
I would suggest to rename to `isDumpingLoadedClassList`

Reply:
Will change.